cli/src/semgrep/rule_match.py

```diff
@@ -205,6 +205,19 @@ def get_ci_unique_key(self) -> Tuple:
             path = self.path
         return (self.rule_id, str(path), self.syntactic_context, self.index)
 
+    def get_path_changed_ci_unique_key(self, rename_dict: Dict[str, Path]) -> Tuple:
+        """
+        A unique key that accounts for filepath renames.
+
+        Results in fewer unique findings than cli_unique_key.
+        """
+        try:
+            path = str(self.path.relative_to(Path.cwd()))
+        except (ValueError, FileNotFoundError):
+            path = str(self.path)
+        renamed_path = str(rename_dict[path]) if path in rename_dict else path
```

Comment:
Wont this only work on the PR that made the rename? Any PR after this would then not be able to do this lookup

Reply:
Discussed offline and agreed this should be ok